<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecieveFileForm</class>
 <widget class="QWidget" name="RecieveFileForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>接收文件:</string>
   </property>
  </widget>
  <widget class="QLabel" name="downloadedLb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1M/100M</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedLb">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10M/S</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="filenameLb">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ABC.EXE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finishBt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>下载完成</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
